### ISSUE TITLE
CIFAR10-Adversarial-Perturbations.ipynb -- Standard rai-toolbox[mushin] install doesn't include dill module

### ISSUE THREAD

**Issue by miscpeeps** [mit-ll-responsible-ai/responsible-ai-toolbox#67]
[CIFAR10-Adversarial-Perturbations.ipynb](https://github.com/mit-ll-responsible-ai/responsible-ai-toolbox/blob/main/examples/CIFAR10-Adversarial-Perturbations.ipynb) example jupyter notebook attempts to load the pretrained models and fails with `ModuleNotFoundError: No module named 'dill'`. Dill module is not included in the standard `rai-toolbox[mushin]` install.

Full error traceback below:
```
ModuleNotFoundError                       Traceback (most recent call last)
Input In [9], in <cell line: 3>()
      1 # Load pretrained model that was trained using a robust approach (i.e., adversarial training)
      2 ckpt_robust = "mitll_cifar_l2_1_0.pt"
----> 3 model_robust = load_model(ckpt_robust)
      4 model_robust.eval();
      6 # Load pretrained model that was trained with standard approach

Input In [7], in load_model(ckpt)
      2 def load_model(ckpt):
----> 3     base_model = load_from_checkpoint(
      4         model = resnet50(),
      5         ckpt = ckpt,
      6         weights_key="state_dict",
      7     )
      9     normalizer = transforms.Normalize(
     10         mean=[0.4914, 0.4822, 0.4465],
     11         std=[0.2023, 0.1994, 0.2010],
     12     )
     14     model = nn.Sequential(normalizer, base_model)

File ~/dev/rai-toolbox-james/responsible-ai-toolbox/src/rai_toolbox/mushin/_utils.py:60, in load_from_checkpoint(model, ckpt, weights_key, weights_key_strip, model_attr)
     57     ckpt = Path.home() / ".torch" / "models" / ckpt
     58 log.info(f"Loading model checkpoint from {ckpt}")
---> 60 ckpt_data: Dict[str, Any] = torch.load(ckpt, map_location="cpu")
     62 if weights_key is not None:
     63     assert weights_key in ckpt_data

File ~/anaconda3/envs/rai-toolbox-james/lib/python3.9/site-packages/torch/serialization.py:713, in load(f, map_location, pickle_module, **pickle_load_args)
    711             return torch.jit.load(opened_file)
    712         return _load(opened_zipfile, map_location, pickle_module, **pickle_load_args)
--> 713 return _legacy_load(opened_file, map_location, pickle_module, **pickle_load_args)

File ~/anaconda3/envs/rai-toolbox-james/lib/python3.9/site-packages/torch/serialization.py:930, in _legacy_load(f, map_location, pickle_module, **pickle_load_args)
    928 unpickler = UnpicklerWrapper(f, **pickle_load_args)
    929 unpickler.persistent_load = persistent_load
--> 930 result = unpickler.load()
    932 deserialized_storage_keys = pickle_module.load(f, **pickle_load_args)
    934 offset = f.tell() if f_should_read_directly else None

File ~/anaconda3/envs/rai-toolbox-james/lib/python3.9/site-packages/torch/serialization.py:746, in _legacy_load.<locals>.UnpicklerWrapper.find_class(self, mod_name, name)
    744     except KeyError:
    745         pass
--> 746 return super().find_class(mod_name, name)

ModuleNotFoundError: No module named 'dill'
```

Installing dill via `pip install dill` in the python environment corrects this error.

**Comment by jgbos** [comment]
Where did you get this model from? The files I have do not require `dill`, the original models from MadryLab do require `dill` and Python 3.6. Btw, you can now find all the models here:

https://github.com/mit-ll-responsible-ai/data_hosting/releases/tag/madrylab_models

**Comment by rsokl** [comment]
Resolved by #69 #73
